src/ip_defer.c

```diff
@@ -20,7 +20,7 @@ struct ip_defer {
  * This is used to inhibit defers if it's the ip deferring code itself causing
  * defer push.
  */
-static unsigned defer_inhibit;
+bool defer_inhibit;
```

Comment:
Declare `defer_inhibit` as `statuc bool`. In addition, you should replace `defer_inhibit = 1` with `defer_inhibit = true`.

Reply:
Done.